src/qt/utilitydialog.cpp

```diff
@@ -20,6 +20,9 @@
 #include "clientversion.h"
 #include "init.h"
 #include "util.h"
+#ifdef ENABLE_WALLET
```

Comment:
> Is there something special about utilitydialog.cpp ?

No, but adding `ENABLE_WALLET` should be avoided everywhere if possible. I think it may be unavoidable here, though.